lib/src/auth.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:flutter/widgets.dart';
+
+/// Dummy authentication service as we only need login details to identify user
+class AmiAppAuth extends ChangeNotifier {
```

Comment:
can we do it without a class? something link anonymous class ?

Reply:
Not really I think 😅 Mainly because `ChangeNotifier` makes it easier for us to identify auth changes and having a class makes it easier to handle/mock sign in/out logic in a single place.